pkg/kapp/clusterapply/delete_change.go

```diff
@@ -56,6 +56,12 @@ func (c DeleteChange) IsDoneApplying() (ctlresm.DoneApplyState, []string, error)
 		return ctlresm.DoneApplyState{Done: true, Successful: true, Message: "Resource orphaned"}, nil, nil
 	}
 
+	if res.IsDeleting() && len(res.Finalizers()) > 0 {
+		return ctlresm.DoneApplyState{Done: false, Message: "waiting on finalizers being removed"},
+			[]string{fmt.Sprintf("Resource deletion step is waiting on following finalizers being removed: %s",
+				strings.Join(res.Finalizers(), ","))}, nil
```

Comment:
can you include example output of how this looks in this thread? we might want to tweak wording a little bit.

Reply:
Yaml File - 
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: mymap
  finalizers:
    - kubernetes
```
Example Output while trying to delete - 
```
Changes

Namespace  Name   Kind       Conds.  Age  Op      Op st.  Wait to  Rs       Ri  
default    mymap  ConfigMap  -       7m   delete  -       delete   ongoing  Deleting  

Op:      0 create, 1 delete, 0 update, 0 noop, 0 exists
Wait to: 0 reconcile, 1 delete, 0 noop

Continue? [yN]: y

4:25:22PM: ---- applying 1 changes [0/1 done] ----
04:25:22PM: info: Resources: TODO resource 'configmap/mymap (v1) namespace: default' is already deleting
4:25:22PM: delete configmap/mymap (v1) namespace: default
4:25:22PM: ---- waiting on 1 changes [0/1 done] ----
4:25:22PM: ongoing: delete configmap/mymap (v1) namespace: default
4:25:22PM: Resource deletion step is waiting on following finalizers being removed: kubernetes
```

Reply:
I'd like to see something like this:

```
4:32:22PM: ongoing: delete configmap/mymap (v1) namespace: default
4:32:22PM:  ^ Waiting on finalizers: kubernetes
```

i believe we have some code already with ` ^ ...` but i dont recall why it wouldnt kick in here.

Reply:
wont this message interleave when multiple resources are awaited?

Reply:
@cppforlife we have code with ` ^ ...` but that is getting used for retryable error and in `buildParentDescMsg` no where else so I have changed the error message using `^` prefix.